### ISSUE TITLE
Bitstreams must always open in new tab

### ISSUE THREAD

**Issue by erpnedir** [DSpace/dspace-angular#3657]
Currently, bitstreams open in the main window if the file size is below the threshold value. It would be better to open it in a new tab.  
Users go back and forth. As the bitstream and metadatas may differ, it would be better to see both at the same time in different tabs.

Some discussion may be seen on mail thread: https://groups.google.com/g/dspace-community/c/6DzqJsv2mcI/m/lmKlvI-xBgAJ

**Comment by alanorth** [comment]
Hi @erpnedir. I noticed this change in behavior when we migrated to DSpace 7. I filed an issue on the backend last year: https://github.com/DSpace/DSpace/issues/8900

**Comment by erpnedir** [comment]
Hi @alanorth I understand that your issue proposes to open the bitstream inline instead of download. We can achive this by configuring a high value to the threshold parameter webui.content_disposition_threshold. 
I offer to open it in browser always in a new tab. The users we talk go back and forth among the pdf and item page. That creates a bad user experience. 
If that behaviour (opening the bitstream on the same page) has any benefit to user, this can be done by a parameter. But if there is no benefit of the user, I think that bitstreams must always open in new tab of browser if the file size is below the threshold value.

**Comment by alanorth** [comment]
@erpnedir Ah yes you are right, I filed that issue over a year ago and forgot that it's slightly different. The bug is with the content-disposition header behaving differently in DSpace 7.

Regarding your suggestion to *always* open bitstreams in a new window, we should get more feedback from other repository administrators to see if their users feel the same. Currently the only links that DSpace opens in a new window are those that are external to the site (for example DOIs or other external links on the item view page).

**Comment by minurmin** [comment]
I think that it should be carefully considered if opening images to new tab should be the default behaviour, even if only for larger files. At least the users should have some visual indication on this.

From general usability/accessibility standpoint (which in EU area _must_ be taken into account because of the European accessibility act), it is usually considered better that users have control on tabs. To quote Nielsen, _it's been a firm web-usability guideline to refrain from [opening new browser windows](https://www.nngroup.com/articles/the-top-ten-web-design-mistakes-of-1999/) for several reasons_ (disorientation, might be confusing to novice or mobile users, prevent use of back button, etc.). See e.g. https://www.nngroup.com/articles/new-browser-windows-and-tabs/

**Comment by MW3000** [comment]
I agree with @minurmin - it's a good practice that only the users themselves open new tabs, not any website on its own. As preferences seem to vary, it might be best to add a configuration.

**Comment by tdonohue** [comment]
I also agree with @minurmin and @MW3000 .  I'm not a fan of opening files in new tabs without users choosing to do so.  I'm not against a configuration, but I'd recommend defaulting to current behavior (which is to open files in the current tab, as long as they are below the threshold in size & a format which the browser understands).

**Comment by mwoodiupui** [comment]
So long as it's a link, not a button-styled-as-link (which should be punishable by law), so I can use the context menu to tell my browser to do what I want, okay.

We have this discussion again and again.  It's a religious question, like which editor is best.  A configurable default would be *minimum* to resolve that.  Every time I wonder whether this should be a user preference, with a site-configured default.

**Comment by alanorth** [comment]
> _So long as it's a link, not a button-styled-as-link (which should be punishable by law), so I can use the context menu to tell my browser to do what I want, okay._

@mwoodiupui I like [what Duke has done](https://hdl.handle.net/10161/32991):

<img width="2720" height="1668" alt="Image" src="https://github.com/user-attachments/assets/bd6ef6fc-7c65-4078-b369-04fc3ec12cdc" />

We often get feedback that the "Files" list is not obvious, and users try to click the thumbnail and it does nothing. Duke's approach above with the very obvious view/download button is nice. I've talked to their team on Slack and they would be happy if that was upstreamed (though not by them likely).

I don't know how they handle multiple files, primary bitstream, etc.

**Comment by mwoodiupui** [comment]
@alanorth I like it too.  The tooltip on the link makes me expect that it handles multiple files with a column of these.  Yes, the plain "Files" section is not nearly clear or prominent enough.
